qubesbuilder/executors/qubes.py

```diff
@@ -171,7 +171,17 @@ def run(  # type: ignore
             # Adjust log namespace
             log.name = f"executor:qubes:{dispvm}"
 
-            # Start the DispVM by copying qubes-builder RPC
+            # Start the DispVM and wait for session being available
+            subprocess.run(
+                [
+                    "/usr/lib/qubes/qrexec-client-vm",
+                    dispvm,
+                    "qubes.WaitForSession",
+                ],
+                stdin=subprocess.DEVNULL,
```

Comment:
Add also capture_output=True, check=True?
Otherwise output will get printed without usual log prefix, and also failure would be silently ignored.